src/test/testRunner.ts

```diff
@@ -13,6 +13,11 @@ import * as fs from 'fs-extra'
  * @param initTests List of relative paths to test files containing root hooks: https://mochajs.org/#available-root-hooks
  */
 export async function runTests(testFolder: string, initTests: string[] = [], testFiles?: string[]): Promise<void> {
+    // Set explicit timezone to ensure that tests run locally do not use the user's actual timezone, otherwise
+    // the test can pass on one persons machine but not anothers. Also, there is a race condition from when we define this
+    // to the first time a Date is used, so this must be defined before Date is used.
+    process.env.TZ = 'US/Pacific'
```

Comment:
can we delete the thing in launchUtilities.ts ?

Reply:
Did some more testing, yeah we should be able to safely drop it. From reading around there may be a race condition, but I don't see it here

Reply:
we could also move it to the module-level in `src/test/testRunner.ts` , to further mitigate a race?